python.json

```diff
@@ -70,5 +70,29 @@
+  {"id": "py.sorted-containers", "imports": ["sortedcontainers"], "name": "Sorted Containers", "repo": "grantjenks/python-sortedcontainers", "tags": ["containers"], "tech": ["data-structures"]},
+  {"id": "py.scoop", "imports": ["scoop"], "name": "SCOOP", "repo": "soravux/scoop", "tags": ["concurrent-ops"], "tech": ["faster-code"]},
+
+  {"id": "py.django-simple-captcha", "imports": ["captcha"], "name": "Django Simple Captcha", "repo": "mbi/django-simple-captcha", "tags": ["captcha"], "tech": ["app-blocks"]},
+  {"id": "py.django-simple-spam-blocker", "imports": [""], "name": "Django Simple Spam Blocker", "repo": "moqada/django-simple-spam-blocker", "tags": ["spam-blocker"], "tech": ["app-blocks"]},
+
+  {"id": "py.audiolLazy", "imports": ["auidolazy"], "name": "AudioLazy", "repo": "danilobellini/audiolazy", "tags": ["real-time-dsp"], "tech": ["computational-science"]},
+  {"id": "py.audioread", "imports": ["audioread"], "name": "audioread", "repo": "beetbox/audioread", "tags": ["audio-files"], "tech": ["audio"]},
+  {"id": "py.beets", "imports": ["beets"], "name": "Beats", "repo": "beetbox/beets", "tags": ["toolkit"], "tech": ["audio"]},
+  {"id": "py.dejavu", "imports": ["dejavu"], "name": "dejavu", "repo": "worldveil/dejavu", "tags": ["audio-recognition"], "tech": ["audio"]},
+  {"id": "py.django-elastic-transcoder", "imports": ["dj_elastictranscoder"], "name": "Django Elastic Transcoder", "repo": "StreetVoice/django-elastic-transcoder", "tags": ["aws-transcoder"], "tech": ["cloud"]},
+  {"id": "py.eyed3", "imports": ["eyed3"], "name": "eyeD3", "repo": "nicfit/eyeD3", "tags": ["id3-tags"], "tech": ["audio"]},
+  {"id": "py.id3reader", "imports": ["id3reader"], "name": "id3reader", "repo": "https://nedbatchelder.com/code/modules/id3reader.py", "tags": ["id3-tags"], "tech": ["audio"]},
+  {"id": "py.m3u8", "imports": ["m3u8"], "name": "m3u8", "repo": "globocom/m3u8", "tags": ["m3u8-files"], "tech": ["media"]},
+  {"id": "py.mingus", "imports": ["mingus"], "name": "mingus", "repo": "bspaans/python-mingus", "tags": ["music"], "tech": ["audio"]},
+  {"id": "py.mutagen", "imports": ["mutagen"], "name": "mutagen", "repo": "quodlibet/mutagen", "tags": ["audio-metadata"], "tech": ["audio"]},
+  {"id": "py.pyaudioanalysis", "imports": ["pyAudioAnalysis"], "name": "pyAudioAnalysis", "repo": "tyiannak/pyAudioAnalysis", "tags": ["audio-analysis"], "tech": ["computational-science"]},
+  {"id": "py.pydub", "imports": ["pydub"], "name": "Pydub", "repo": "jiaaro/pydub", "tags": ["toolkit"], "tech": ["audio"]},
+  {"id": "py.pyechonest", "imports": ["pyechonest"], "name": "Pyechonest", "repo": "echonest/pyechonest", "tags": ["music-search"], "tech": ["api"]},
+  {"id": "py.timeside", "imports": ["timeside"], "name": " TimeSide", "repo": "Parisson/TimeSide", "tags": ["web-audio"], "tech": ["audio"]},
```

Comment:
tags: maybe toolkit? agreed, it's a huge package, but web-audio sounds too common

Reply:
but it has web in it. i can do web-toolkit. will it work? but web-audio is more precise. it's tools to make audio for web.

Reply:
would it be ok to keep them as two separate tags like [ "web", "toolkit" ]